src/index.ts

```diff
@@ -369,33 +280,4 @@ export async function prepareCache({
 	return cacheFiles;
 }
 
-function findCargoToml(
-	files: BuildOptions["files"],
-	entrypoint: BuildOptions["entrypoint"]
-) {
-	let currentPath = path.dirname(entrypoint);
-	let cargoTomlPath;
-
-	// eslint-disable-next-line no-constant-condition
-	while (true) {
-		cargoTomlPath = path.join(currentPath, "Cargo.toml");
-		if (files[cargoTomlPath]) break;
-		const newPath = path.dirname(currentPath);
-		if (currentPath === newPath) break;
-		currentPath = newPath;
-	}
-
-	return cargoTomlPath;
-}
-
-export const getDefaultCache = ({ files, entrypoint }: BuildOptions) => {
```

Comment:
Did Zeit change their caching mechanism, or are we just giving up since it never seemed to work?

Reply:
I belive they expect [`prepareCache`](https://github.com/zeit/now/blob/master/DEVELOPING_A_RUNTIME.md#preparecache) for the caching and do not expect `getDefaultCache`, so this seemed to be redundant, as `now-rust` also exposes `prepareCache`. I could have missed it somewhere, but a quick search on [`zeit/now`](https://github.com/zeit/now/search?q=getDefaultCache&unscoped_q=getDefaultCache) did not return results for `getDefaultCache`.

I think the caching could be improved though, as while locally caching seems to work, when deploying multiple functions the build seems to re-download and build cargo packages, so builds can take quite long. That does not really have a negative effect on the development experience and I don't think zeit's pricing model is based on build time, so I don't think it's anything urgent, just for the future.